### PR TITLE
First stab at a helm chart

### PR DESCRIPTION
**Pull request by mothershipper**

### CODE REVIEW

File: charts/rds-auth-proxy/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+A two-layer proxy for connecting into RDS postgres databases based on IAM authentication.
+
+**Homepage:** <https://github.com/mothership/rds-auth-proxy>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| Jack Wink |  | https://github.com/mothership/rds-auth-proxy/issues |
```
Comment (jonathan-mothership) [inline charts/rds-auth-proxy/README.md]: could put your Github email

Comment (mothershipper) [reply charts/rds-auth-proxy/README.md]: nah, I'd rather direct to GitHub, we have our security policy and other stuff there

File: charts/rds-auth-proxy/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Mothership
```
Comment (jonathan-mothership) [inline charts/rds-auth-proxy/LICENSE]: Mothership Technologies, Inc.

File: charts/rds-auth-proxy/templates/cert.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{{- if .Values.proxy.certManager.enabled }}
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  name: {{ include "app.clientCertName" . }}
+spec:
+  dnsNames:
+    - {{ include "app.fullname" . }}.{{ .Release.Namespace }}.svc
+    - {{ include "app.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
```
Comment (jonathan-mothership) [inline charts/rds-auth-proxy/templates/cert.yaml]: This assumes cluster domain name is default of `cluster.local`.

File: charts/rds-auth-proxy/templates/service_account.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+kind: ServiceAccount
+metadata:
+  name: {{ template "app.fullname" . }}
+  annotations:
+    {{- with .Values.serviceAccount.annotations }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+{{- with .Values.serviceAccount.imagePullSecrets }}
```
Comment (jonathan-mothership) [inline charts/rds-auth-proxy/templates/service_account.yaml:12]: > [ERROR] templates/: template: rds-auth-proxy/templates/service_account.yaml:12:16: executing "rds-auth-proxy/templates/service_account.yaml" at <.Values.serviceAccount.imagePullSecrets>: map has no entry for key "imagePullSecrets"

**Review by jonathan-mothership — APPROVED** [review]
Small nits

File: charts/rds-auth-proxy/README.md (hunk 1 above)
Comment (jonathan-mothership) [reply charts/rds-auth-proxy/README.md]: Update to `@mothershipper`?

File: charts/rds-auth-proxy/values.yaml [hunk 5]
```diff
@@ -58,6 +58,8 @@ serviceAccount:
   # -- Annotations for the service-account - this can be used for IRSA
```
Comment (jonathan-mothership) [inline charts/rds-auth-proxy/values.yaml]: Camelcase instead of kebab-case

File: charts/rds-auth-proxy/README.md (hunk 1 above)
Comment (mothershipper) [reply charts/rds-auth-proxy/README.md]: had to update name to mothershipper -- lint yelled at me

File: charts/rds-auth-proxy/templates/cert.yaml (hunk 3 above)
Comment (mothershipper) [reply charts/rds-auth-proxy/templates/cert.yaml]: what would the suggested resolution be?

Comment (jonathan-mothership) [reply charts/rds-auth-proxy/templates/cert.yaml]: Add a `clusterName` to the values file and interpolate here.

File: charts/rds-auth-proxy/values.yaml (hunk 5 above)
Comment (mothershipper) [reply charts/rds-auth-proxy/values.yaml]: just removing the hyphen, this is for docs

**Pull request merged**